### ISSUE TITLE
FormGotoCommit: Shift+Tab does not focus all controls (in contrast to Tab)

### ISSUE THREAD

**Issue by mstv** [gitextensions/gitextensions#11226]
### Environment

- Git Extensions 33.33.33
- Build 04dba961a1383a94a015b4ed54d8d1cead043510
- Git 2.42.0.windows.1
- Microsoft Windows NT 10.0.19045.0
- .NET 6.0.22
- DPI 96dpi (no scaling)
- Portable: False
- Microsoft.WindowsDesktop.App Versions

```
    Microsoft.WindowsDesktop.App 6.0.22 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

### Issue description

FormGotoCommit lacks mnemonics.
`Shift+Tab` focuses less controls than `Tab`: namely `Go to branch` and `Go to tag`.

### Steps to reproduce

- Hit `Shift+Ctrl+G`
- Repeat `Shift+Tab`
- Repaat `Tab`

### Did this work in previous version of GitExtensions?

unlikely

**Comment by RussKie** [comment]
How peculiar...
This form needs a layout refresh, perhaps this can be fixed alongside.
